CHANGELOG.md

```diff
@@ -1,19 +1,26 @@
 # laminr v1.0.0
 
-This is a major rewrite of the package, resulting in many breaking changes but also offering an interface that is much more consistent with the Python API. Please see the updated ["Get started"](https://laminr.lamin.ai/articles/laminr.html) vignette for more information.
+This is a major rewrite of the package to offer an interface that has feature-parity with the Python API (PR #146).
 
-## BREAKING CHANGES
+Please see the updated ["Get started"](https://laminr.lamin.ai/articles/laminr.html) vignette for more information.
+
+Migration guide:
```

Comment:
@lazappi -- I accepted all your suggestions.

To confirm: Do we really only need a single bullet under "Migration guide"?

What does "largely similar" mean? What's the difference?

Reply:
Rather than doing the re-wording, can you list _what_ changed so that people can update their code and get intuition which calls are differing?

Ideally you make a table

<v1 | >=v1
--- | ---
`hi()` | `hello()`

Reply:
This is what we did for the lamindb v1 migration:

<img width="882" alt="image" src="https://github.com/user-attachments/assets/214367dd-70a1-4555-955a-2de943af6a0a" />

Reply:
Added a suggestion with a table of changes from the "Get started" vignette